examples/tutorials/vectors.py

```diff
@@ -0,0 +1,486 @@
+
+fig = pygmt.Figure()
+# Vector 1 with default unit as cm
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X5i/5i",
+    frame="ag",
+    x=2,
+    y=8,
+    style="v1c+e",
+    direction=[[0], [3]],
+    pen="2p",
+    color="red3",
+)
+# Vector 2 after changing default unit to in
+with pygmt.config(PROJ_LENGTH_UNIT="i"):
+    fig.plot(
+        x=2,
+        y=7,
+        direction=[[0], [3]],
+        style="v1c+e",
+        pen="2p",
+        color="red3",
+    )
+fig.show()
+
+########################################################################################
+# Vectors can also be plotted by including all the information
+# about a vector in a single list. However, this requires creating
+# a 2D list or numpy array containing all vectors.
+# Each vector list contains the information structured as:
+# ``[x_start, y_start, direction_degrees, length]``
+#
+# If this approach is chosen, the ``data`` parameter must be
+# used instead of ``x``, ``y`` and  ``direction``.
+
+# Create a list of lists that include each vector information
+vectors = [[2, 3, 45, 4]]  # vectors structure: [[ 2  3 45  4]]
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X10c/10c",
+    frame="ag",
+    data=vectors,
+    style="v0.6c+e",
+    pen="2p",
+    color="red3",
+)
+fig.show()
+
+########################################################################################
+# Using the functionality mentioned in the previous example,
+# multiple vectors can be plotted at the same time. Another
+# vector could be simply added to the 2D list or numpy
+# array object
+# and passed using ``data`` parameter.
+
+# vector specifications structured as: [x_start, y_start, direction_degrees, length]
+vector_1 = [2, 3, 45, 4]
+vector_2 = [7.5, 8.3, -120.5, 7.2]
+# Create a list of lists that include each vector information
+vectors = np.array([vector_1, vector_2])
+# vectors structure:
+# [[   2.     3.    45.     4. ]
+#  [   7.5    8.3 -120.5    7.2]]
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X10c/10c",
+    frame="ag",
+    data=vectors,
+    style="v0.6c+e",
+    pen="2p",
+    color="red3",
+)
+fig.show()
+
+########################################################################################
+# In this example, cartesian vectors are plotted over a Mercator
+# projection of the continental US. The x values represent the
+# longitude and y values represent the latitude where the vector starts.
+#
+# This example also shows some of the styles a vector supports.
+# ``+ba`` specifies that the begining point of the vector ``+b``
+# should take the shape of a circle ``c``. Similarly, the end
+# point of the vector ``+e`` should have an arrow shape ``a``
+# (to draw a plain arrow, use ``A`` instead). Lastly, the ``+a``
+# specifies the angle of the vector head apex (30 degrees in
+# this example).
+#
+# More styling options can be found here
```

Comment:
How about putting this paragraph at the end of the one above?

Besides, I'd like to suggest also adding a link to [Vector Attributes](https://docs.generic-mapping-tools.org/latest/plot.html#vector-attributes).

```
:gmt-docs:`Vector Attributes <plot.html#vector-attributes>`
```

Reply:
I suggest removing this sentence because we already mentioned it before.

> Besides, I'd like to suggest also adding a link to [Vector Attributes](https://docs.generic-mapping-tools.org/latest/plot.html#vector-attributes).
> 
> ```
> :gmt-docs:`Vector Attributes <plot.html#vector-attributes>`
> ```

I prefer not to add the link.

Reply:
I'm not sure if you came to a decision about this. I haven't made any changes to this yet, so please let me know what you want us to do. Removing the link makes the most sense to me because it was already mentioned before, but I'm fine with whatever you decide.

Reply:
Yes, let's remove it.